src/resample.c

```diff
@@ -74,6 +75,48 @@ main (int argc, char *argv [])
 		{	k ++ ;
 			converter = parse_int_or_die (argv [k], "converter") ;
 			}
+		else if (strcmp (argv [k], "-r") == 0)
+		{   k ++ ;
+            if (sscanf (argv [k], "%d,%d,%c,%d", &sfinfo.samplerate, &sfinfo.channels, &raw_type, &raw_bits) != 4)
+            {   printf ("Error : unable to decode raw audio parameters.\n") ;
+                usage_exit (argv [0]) ;
+                }
+            sfinfo.format = SF_FORMAT_RAW;
+            if (raw_type == 'i' && ( raw_bits == 8 || raw_bits == 16 || raw_bits == 24 || raw_bits == 32 ) )
+            {   switch (raw_bits)
+                {   case  8:
+                        sfinfo.format |= SF_FORMAT_PCM_S8;
+                        break;
+
+                    case 16:
+                        sfinfo.format |= SF_FORMAT_PCM_16;
+                        break;
+
+                    case 24:
+                        sfinfo.format |= SF_FORMAT_PCM_24;
+                        break;
+
+                    case 32:
+                        sfinfo.format |= SF_FORMAT_PCM_32;
+                        break;
+                    }
+                }
+            else if (raw_type == 'f' && ( raw_bits == 32 || raw_bits == 64 ) )
+            {   switch (raw_bits)
+                {   case 32:
+                        sfinfo.format |= SF_FORMAT_FLOAT;
+                        break;
+
+                    case 64:
+                        sfinfo.format |= SF_FORMAT_DOUBLE;
+                        break;
+                    }
+                }
+		    else
```

Comment:
Weird indentation in that `else`.

Reply:
it's due to mixture of tabs/spaces, I bet